packages/templates/nextjs/.env.example

```
@@ -1,4 +1,6 @@
 DODO_PAYMENTS_API_KEY=your-api-key
 DODO_PAYMENTS_WEBHOOK_KEY=your-webhook-secret
 DODO_PAYMENTS_RETURN_URL=https://yourdomain.com/checkout/success
-DODO_PAYMENTS_ENVIRONMENT="test"or"live"
+DODO_PAYMENTS_ENVIRONMENT="test"or"live"
+
+STRIPE_SECRET_KEY=your=stripe-secret-key
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)